### DISCUSSION TITLE
Баг

### DISCUSSION THREAD

**Discussion by Lenivi** [MrZillaGold/VK2Discord#208] [Общее]
Когда в посте содержится ссылка на статью. То под плашкой "Вложения" выводятся странная ссылка, но переходя по ней всё хорошо. Но всё же выглядит это, не очень красиво.
![image](https://user-images.githubusercontent.com/86191624/122682219-9ae66700-d211-11eb-9414-391f941b55b4.png)

P.S Не знаю куда можно сообщать о багах, поэтому пишу вот сюда.

**Comment by egorprnn** [comment]
Уже обсуждалось: https://github.com/MrZillaGold/VK2Discord/issues/80.

Об ошибках можно сообщать [здесь](https://github.com/MrZillaGold/VK2Discord/issues).
